spa/src/rest-interfaces

```diff
@@ -0,0 +1 @@
+../../src/rest-interfaces
```

Comment:
This is a linux symbolic link. I hope the production pipeline (linux) will recognise this as well as other developer's machine.